zerver/tests/test_message_dict.py

```diff
@@ -644,3 +644,33 @@ def test_sew_messages_and_reaction(self) -> None:
             self.assertEqual(data["reactions"][0]["emoji_name"], "simple_smile")
             self.assertTrue(data["id"])
             self.assertTrue(data["content"])
+
+    def test_reaction_order(self) -> None:
+        sender = self.example_user("othello")
+        receiver = self.example_user("hamlet")
+        recipient = Recipient.objects.get(type_id=receiver.id, type=Recipient.PERSONAL)
+        sending_client = make_client(name="test suite")
+
+        message = Message(
+            sender=sender,
+            recipient=recipient,
+            content="whatever",
+            date_sent=timezone_now(),
+            sending_client=sending_client,
+            last_edit_time=timezone_now(),
+            edit_history="[]",
+        )
+        message.set_topic_name("whatever")
+        message.save()
```

Comment:
I think we should just use the `self.send_stream_message` helper to generate the message.  Generally we try to avoid just creating custom database objects like this, since they usually violate various invariants of the codebase.

Reply:
This stuff was copy pasted from the other test here - looks like both tests could benefit from this improvement.
My test *certainly* violates invariants of the codebase - and was looking for something like this.

Might send another diff to update the other test here (`test_sew_messages_and_reaction`) to use the helper

Reply:
Yeah, generally when I notice things like this, I'll just add an extra commit in the branch to clean up the existing test.  

There's a related series of testing sweeps I want to do of things like `realm.save` => `do_set_realm_property` (and the same for the other common object types, like Message here), so that there won't be existing tests with this sort of pattern to copy-paste from.  Could be a good little follow-up project I suppose :)